docs/preview/connected-registry/quickstart-send-connected-registry-events-to-event-grid.md

```diff
@@ -0,0 +1,175 @@
+
+In this tutorial, we configure the connected registry to send artifact events like push and delete to Azure Event Grid in a two step process.
+
+1. Configure connected registry to generate events on patterns of interest.
+2. Configure Azure Event Grid to subscribe and filter for events generated by connected registry.
+
+## Notification Events Generation on a connected registry
+
+Use the following CLI command to configure the connected registry to generate events for all artifacts
+
+```azurecli
+az acr update -n $ACR_NAME
+  --name $ACR_CONNECTED_REGISTRY_NAME \
+  --add-notifications *:*
+```
+
+If you are interested in generating events only for specific artifact patterns or specific actions, you can specify the patterns in the form shown below
+
+```
+az acr update -n $ACR_NAME \
+  --name $ACR_CONNECTED_REGISTRY_NAME \
+  --add-notifications [PATTERN1 PATTERN2 ...]
+```
+
+where each pattern is defined as
+
+  ```
+imageName ":" action
+  ```
+
+The `imageName` and `action` are further defined as
+
+  ```yaml
+imageName: name [ ":" tag ] [ "@" digest ]
+action: "push" or "delete" or "*"
+  ```
+
+The wildcard `*` should be used to cover all values on a section. 
+
+For example,
+
+1. \*:\* - Notify all actions on all tags and digests on all artifacts on the connected registry
+2. \*:\*:\*- Notify all actions on all tags on all artifacts on the connected registry
+3. \*@\*:\*- Notify all actions on all digests on all artifacts on the connected registry
+4. path/to/repo:\* - Notify all actions on all artifacts on repository '/path/to/repo' on the connected registry
+5. path/to/repo:myTag:\* - Notify all actions on tag 'myTag' on repository '/path/to/repo' on the connected registry
+6. path/to/repo@myDigest:delete - Notify only 'delete' action on digest 'myDigest' on repository '/path/to/repo' on the connected registry
+7. path/to/repo/\*:push - Notify only 'push' action on all artifacts on all repositories that match '/path/to/repo/\*' on the connected registry
+
+## Event Grid Subscription and filtering Connected Registry Events on Event Grid
+
+In this step, you will configure eventgrid to subscribe for events from connected registry. You can use the `--advanced-filter` on event grid to filter out connected registry specific events. The connected registry events will align to the existing Azure Container Registry event types `Microsoft.ContainerRegistry.ImagePushed` and `Microsoft.ContainerRegistry.ImageDeleted`. 
+
+Before proceeding, fetch the Azure Container Registry Id and the event endpoint for your events.
+
+```
+ACR_REGISTRY_ID=$(az acr show --name $ACR_NAME --query id --output tsv)
+APP_ENDPOINT=<your-app-endpoint>
```

Comment:
Add a comment indicating that this corresponds to the app endpoint created in a separate tutorial